website/screens/components/table/code/TableCodePage.tsx

```diff
@@ -66,6 +91,12 @@ const sections = [
           </>
         ),
       },
+      {
+        title: "Reduced usage",
+        content: (
+          <p>This example is not available yet, it will be added soon.</p>
+        ),
+      },
```

Comment:
I am not sure if this is suitable for the examples or must be only placed in 'Usage'